frontend/public/css/style.css

```diff
@@ -225,6 +234,13 @@ input[type=range]:focus {
 }
 
 /* Preset area configuration. */
+#tagsearch {
+	width: 100%;
```

Comment:
Please watch the indentation of the CSS files as they may become messy if not properly indented.